### ISSUE TITLE
Filesafe doesn't work in the iOS app

### ISSUE THREAD

**Issue by tmoravec** [standardnotes/filesafe-bar#3]
It works in the web version in Safari but not in the app (iOS Safari is worth testing as well but that's a story for another day). The Filesafe pane is not there, and it's not in the left or right drawers either.

Happens on both iPhone and iPad. Latest iOS 12.2.

**Comment by atmoio** [comment]
Right, the mobile app doesn't support editor stack extensions yet. Only editors and themes are supported on mobile. I imagine we'll support them at some point in the future, but TBD.
